README.md

```diff
@@ -156,14 +161,6 @@ $ cp www/json/startupConfig.json.sample www/json/startupConfig.json
 $ cp ..... www/json/connectionConfig.json
 ```
 
-### Activation (after install, and in every new shell)
-
-```
-$ source setup/activate_native.sh
-```
-
-### Activation (after install, and in every new shell)
-
```

Comment:
why did you remove this? We do need to activate after install.

Reply:
I was able to build the app successfully. Let me verify the above again just to be sure.

Reply:
were you able to build the app starting in a new terminal and following the instructions in the README exactly? 
When you check, please show, don't tell.